acs-integration-tests/src/test/java/com/ge/predix/test/utils/ZacOptionalTestSetup.java

```diff
@@ -0,0 +1,94 @@
+package com.ge.predix.test.utils;
+
+import static com.ge.predix.test.utils.ACSTestUtil.isServerListening;
+
+import java.net.URI;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.testng.SkipException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ZacOptionalTestSetup implements OptionalTestSetup {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZacOptionalTestSetup.class);
+
+    @Autowired(required = false)
+    private OAuth2RestTemplate zacRestTemplate;
+
+    @Value("${ZAC_URL}")
+    private String zacUrl;
+
+    @Value("${ACS_SERVICE_ID:predix-acs}")
+    private String serviceId;
+
+    @Value("${acs.zone.header.name}")
+    private String acsZoneHeaderName;
```

Comment:
Zone header name is hardcoded to "Predix-Zone-Id" everywhere else.  We should pick a consistent approach: either hardcode here or switch to using this construct  everywhere else.